src/HelloWorld.java

```diff
@@ -0,0 +1,37 @@
+package com.example.helloworld;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class HelloWorld {
+   public static void main(String[] args) throws InterruptedException {
+        String valueDriverSetProperty = "webdriver.chrome.driver";
+        String pathSetProperty = "/Users/ilsemacias/Desktop/drivers/chromedriver";
+        String url = "http://www.practiceselenium.com/";
+
+        //Declaration and instance of drivers.
+        //Setup
+        System.setProperty(valueDriverSetProperty, pathSetProperty);
+        WebDriver driver = new ChromeDriver();
+        driver.manage().deleteAllCookies();
+        driver.get(url);
+        Utilities.printMessage("The URL is: "+ url);
```

Comment:
Me gusta que apliques la practica de imprimir lo que sucede en la prueba, te recomiendo que uses esta herramienta para hacer logs llamada log4j2 https://logging.apache.org/log4j/2.x/
Tiene mas funcionalidades y declaras tu intencion de mantener un registro sobre el comportamiento de la prueba. Al igual que mi comentario abajo sobre los comments, evita el exceso, usalo para bloques de codigo en el que puedas explicar en pocas palabras que sucede ahi, ejemplo "Step 1: Access the homepage and click the herbal button"

Reply:
Me agrada la idea, pero me gustaría hacer eso en otro PR que sea para agregar log4j.